users/spidey3/layer_rgb.c

```diff
@@ -0,0 +1,84 @@
+#include QMK_KEYBOARD_H
+
+#include <spidey3.h>
+#include <velocikey.h>
+
+extern rgblight_config_t rgblight_config;
```

Comment:
You shouldn't need this function, and it should be removed.  Any of the info that you might need can be gotten via this functions: 
https://docs.qmk.fm/#/feature_rgblight?id=query

Reply:
Removed.